service/feeder/live.go

```diff
@@ -0,0 +1,140 @@
+package feeder
+
+import (
+	"bytes"
+	"fmt"
+	"sync"
+
+	"github.com/fxamacker/cbor/v2"
+	"github.com/klauspost/compress/zstd"
+	"github.com/optakt/flow-dps/models/dps"
+	"github.com/rs/zerolog"
+	"go.nanomsg.org/mangos/v3"
+	"go.nanomsg.org/mangos/v3/protocol/sub"
+
+	// register transports
+	_ "go.nanomsg.org/mangos/v3/transport/all"
```

Comment:
In those cases, I put the comment behind the import so we don't pollute the lines.